packages/mobile/src/redux/sagas.ts

```diff
@@ -77,6 +77,6 @@ export function* rootSaga() {
   yield spawn(dappKitSaga)
   yield spawn(feesSaga)
   yield spawn(localCurrencySaga)
-  yield spawn(web3Saga)
   yield spawn(sentrySaga)
+  yield spawn(web3Saga)
```

Comment:
Is this important for the fix?

Reply:
It conceptually makes sense to wait til the other sagas are set up before we run web3 b.c. that saga will check the sync status. I moved it while debugging. Not 100% sure if it's *required* for the fix though.

Reply:
The required parts of a combination of adding a spawn(waitForWeb3) line to the web3 saga and removing the await waitforweb3 line from the getToken utility